.github/workflows/build_all.yml

```diff
@@ -107,127 +107,6 @@ jobs:
         run: |
           pip install artifactory
           python ci-scripts/helm_repo_rotation.py
-  agw-build:
```

Comment:
Please remove the reference to `agw-build` from line 12

Reply:
Nice catch! Changed it.